src/app.rs

```diff
@@ -509,11 +512,12 @@ impl cosmic::Application for CosmicLauncher {
                     cosmic::app::message::app(Message::Hide)
                 });
             }
-            Message::AltTab => {
-                if self.alt_tab {
-                    self.focus_next();
+            Message::AltTab { reverse } => {
+                self.alt_tab = true;
```

Comment:
Based on my comment in the conversations section, I think we should switch this back to not cycle when opened for now.
Once we have the applications ordered by most recently focused, we should make the change to cycle on the first use of Alt-Tab.